frontend/src/components/home/WorkoutCard.tsx

```diff
@@ -1,10 +1,30 @@
 import {TypeWorkoutCard} from "../../interfaces/types.ts";
 import "./WorkoutCard.css";
+import {useNavigate} from "react-router-dom";
+import React from "react";
 
 function WorkoutCard (props: TypeWorkoutCard) {
+    const navigate = useNavigate();
+
+    function handleClick():void {
+        navigate(`/workout/${props.workout.id}`);
+    }
+
+    function handleKeyDown(event:React.KeyboardEvent):void {
+        if(event.key === "Enter" || event.key === " ") {
+            navigate(`/workout/${props.workout.id}`);
+        }
+    }
 
     return (
-        <section className={"workout-card"}>
+        <section
+            role={"button"}
+            onClick={handleClick}
+            onKeyDown={handleKeyDown}
+            tabIndex={0}
+            aria-label={"Go to workout details"}
+            className={"workout-card"}
+        >
```

Comment:
Bad practice. Wenn ich in der übersicht bin, wo ich alle workouts sehe und enter drücke, auf welches workout komme ich dann?

Reply:
Auf keins? Man muss sich doch mit tab durchklicken. Ich füge noch ein aria-labelledby hinzu